lib/api/apiUtils/object/versioning.js

```diff
@@ -142,44 +188,98 @@ export function versioningPreprocessing(bucketName, bucketMD, objectKey,
             if (nullVersionId === undefined) {
                 return callback(null, options);
             }
-            options.deleteNullVersionData = true;
             return _deleteNullVersionMD(bucketName, objectKey,
-                { versionId: nullVersionId }, log,
-                err => callback(err, options));
+                { versionId: nullVersionId }, log, (err, nullDataToDelete) => {
+                    if (err) {
+                        process.stdout.write('====================\n');
```

Comment:
Is the console output useful (isn't the log enough)? Particularly since we consider `errors.NoSuchKey` to be a legit error.